coretex/networking/network_response.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, response: Response, endpoint: str):
 
         try:
             self.json = response.json()
-        except ValueError:
+        except (ValueError, RuntimeError) as e:
```

Comment:
Add description on why `RuntimeError` was added here. Why is `e`  declared here? It is unused. Just do `except ValueError, RuntimeError:`

Reply:
description is still not missing